src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -1,13 +1,16 @@
 // eslint-disable-next-line no-use-before-define
 import * as React from 'react';
 import ContextMenu from 'react-native-context-menu-view';
-import {Animated, Text, View} from 'react-native';
+import {Animated, Text, TouchableOpacity, View} from 'react-native';
 import {Swipeable} from 'react-native-gesture-handler';
 import {useAnimatedStyle, useSharedValue} from 'react-native-reanimated';
+import FastImage from 'react-native-fast-image';
 
+import {useNavigation} from '@react-navigation/core';
 import ChatReplyView from './ChatReplyView';
 import IconChatCheckMark from '../../../assets/icon/IconChatCheckMark';
 import IconChatClockGrey from '../../../assets/icon/IconChatClockGrey';
+import IconVideoPlay from '../../../assets/icon/IconVideoPlay';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new imports added in this section seem to be necessary for the new features being implemented, such as handling attachments and video play icons. The `TouchableOpacity` import is used for making the attachments clickable, and `FastImage` is used for efficient image loading. The `useNavigation` hook from `@react-navigation/core` is used to navigate to the video viewer screen when a video attachment is clicked.
```diff
+ import {Animated, Text, TouchableOpacity, View} from 'react-native';
+ import FastImage from 'react-native-fast-image';
+ import {useNavigation} from '@react-navigation/core';
+ import IconVideoPlay from '../../../assets/icon/IconVideoPlay';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->